tests/rptest/clients/rpk.py

```diff
@@ -1889,7 +1888,8 @@ def alter_cluster_quotas(self,
                              default=[],
                              name=[],
                              dry=False,
-                             output_format="json"):
+                             output_format="json",
+                             node: Optional[ClusterNode] = None):
```

Comment:
This isn't passed through to `_run_cluster_quotas`

Reply:
Good spot! Fixed